clojure-ts-mode.el

```diff
@@ -746,34 +761,129 @@ The possible values for this variable are
+    ("let"             . (:block 1))
+    ("letfn"           . (:block 1))
+    ("locking"         . (:block 1))
+    ("loop"            . (:block 1))
+    ("match"           . (:block 1))
+    ("ns"              . (:block 1))
+    ("struct-map"      . (:block 1))
+    ("testing"         . (:block 1))
+    ("when"            . (:block 1))
+    ("when-first"      . (:block 1))
+    ("when-let"        . (:block 1))
+    ("when-not"        . (:block 1))
+    ("when-some"       . (:block 1))
+    ("while"           . (:block 1))
+    ("with-local-vars" . (:block 1))
+    ("with-open"       . (:block 1))
+    ("with-precision"  . (:block 1))
+    ("with-redefs"     . (:block 1))
+    ("defrecord"       . (:block 2))
+    ("deftype"         . (:block 2))
+    ("are"             . (:block 2))
+    ("as->"            . (:block 2))
+    ("catch"           . (:block 2))
+    ("condp"           . (:block 2))
+    ("bound-fn"        . (:inner 0))
+    ("def"             . (:inner 0))
+    ("defmacro"        . (:inner 0))
+    ("defmethod"       . (:inner 0))
+    ("defmulti"        . (:inner 0))
+    ("defn"            . (:inner 0))
+    ("defn-"           . (:inner 0))
+    ("defonce"         . (:inner 0))
+    ("deftest"         . (:inner 0))
+    ("fdef"            . (:inner 0))
+    ("fn"              . (:inner 0))
+    ("reify"           . (:inner 0))
+    ("use-fixtures"    . (:inner 0)))
+  "Default semantic indentation rules.
+
+The format reflects cljfmt indentation rules.  All the default rules are
+aligned with
+https://github.com/weavejester/cljfmt/blob/0.13.0/cljfmt/resources/cljfmt/indents/clojure.clj")
+
+(defun clojure-ts--match-block-0-body (bol first-child)
+  "Match if expression body is not at the same line as FIRST-CHILD.
+
+If there is no body, check that BOL is not at the same line."
+  (let* ((body-pos (if-let* ((body (treesit-node-next-sibling first-child)))
+                       (treesit-node-start body)
+                     bol)))
+    (< (line-number-at-pos (treesit-node-start first-child))
+       (line-number-at-pos body-pos))))
+
+(defun clojure-ts--node-pos-match-block (node parent bol block)
+  "Return TRUE if NODE index in the PARENT matches requested BLOCK.
+
+NODE might be nil (when we insert an empty line for example), in this
+case we look for next available child node in the PARENT after BOL
+position.
+
+The first node in the expression is usually an opening paren, the last
+node is usually a closing paren (unless some automatic parens mode is
+not enabled).  If requested BLOCK is 1, the NODE index should be at
+least 3 (first node is opening paren, second node is matched symbol,
+third node is first argument, and the rest is body which should be
+indented.)"
+  (if node
+      (> (treesit-node-index node) (1+ block))
+    (when-let* ((node-after-bol (treesit-node-first-child-for-pos parent bol)))
+      (> (treesit-node-index node-after-bol) (1+ block)))))
+
+(defun clojure-ts--match-form-body (node parent bol)
```

Comment:
I think you forgot to add a docstring here. (`checkdoc` complains for docstrings for all functions, not just the public ones)

Reply:
Forgot to add :) Now fixed, thanks!